### ISSUE TITLE
Analysis report part 2 feedback

### ISSUE THREAD

**Issue by ugurdogrusoz** [Ali-Sabbagh/CS319-2.E#3]
- Requirements and UC model are complete; however, I find your game a little too simple! You could enrich it by adding some bonuses or other types of game objects.
- Object model could be better
- Dynamic model could be better (only 2 non-straightforward SDs)
- UI design is nice but missing the key ingame play screen
Details:
[2E.analysis.2.pdf](https://github.com/Ali-Sabbagh/CS319-2.E/files/852349/2E.analysis.2.pdf)

**Comment by 5abbagh** [comment]
Is it possible to make the changes stated above and re-upload the Analysis Report?
Or should they appear in the Design Report?

**Comment by ugurdogrusoz** [comment]
Yes, please revise your report and upload with a different name! Revision to the analysis report doesn't belong in Design report.

**Comment by 5abbagh** [comment]
Hocam, changes and revision to our Analysis report were made as requested.
The new file is also uploaded to the repository.
Best regards,

[2E.Analysis Final Revision.pdf](https://github.com/Ali-Sabbagh/CS319-2.E/files/894424/2E.Analysis.Final.Revision.pdf)
